src/commands.ts

```diff
@@ -1748,3 +1748,82 @@ export const autoComplete = (
   }
   return false
 }
+export const canIndent = () => (state: ManuscriptEditorState) => {
+  const { $from } = state.selection
+  const nodeType = $from.node().type
+
+  // We should also add paragraph once the implementation done
+  const allowedNodeTypes = [schema.nodes.section_title]
+  if (!allowedNodeTypes.includes(nodeType)) {
+    return false
+  }
+
+  // Check if the selection is inside the body
+  const isBody = hasParentNodeOfType(schema.nodes.body)(state.selection)
+  if (!isBody) {
+    return false
+  }
+
+  return true
+}
+
+export const indent = () => (state: EditorState) => {
+  const { $from } = state.selection
+  const nodeType = $from.node().type
+  if (nodeType === schema.nodes.section_title) {
+    moveSectionToSubsection()
```

Comment:
we need to call the second function, or keep it as one function

Reply:
so the indent button will not have any effect! `moveSectionToSubsection()` returns just a function and will not execute it's implementation!

Reply:
Oh 😮,  it seems I missed that in my last refinement. Thank you for catching it!